README.md

```diff
@@ -792,10 +792,16 @@ Boolean to declare whether the service's init script has a functional status com
 
 ssh_key_ensure
 --------------
-Export node SSH key. Valid values are 'present' and 'absent'.
+Export node SSH key. Valid values are 'present' and 'absent'. Requires ssh_key_export set to 'true'.
 
 - *Default*: 'present'
 
+ssh_key_export
+--------------
+Manage node SSH key with exported resources. Valid values are 'true' and 'false'.
```

Comment:
ssh_key_export does nearly the same as ssh_key_ensure. Any suggestion on how to better clarify the difference between them?